src/components/ReportActionItem/MoneyRequestAction.js

```diff
@@ -94,26 +92,27 @@ const MoneyRequestAction = (props) => {
 
     return (
         <>
-            <IOUQuote
-                action={props.action}
+            <IOUPreview
+                iouReportID={props.requestReportID}
                 chatReportID={props.chatReportID}
+                isBillSplit={hasMultipleParticipants}
+                isIOUAction
```

Comment:
This is implicitly setting `isIOUAction` to `true` right? Isn't that the default prop set in IOUPreivew anyways so probably not needed?